packages/@azure/keyvault/lib/index.ts

```diff
@@ -0,0 +1,6 @@
+import { RestError } from '@azure/ms-rest-js';
+
+export * from "./secretsClient";
+export * from "./secretsModels";
+export * from "./clientOptions";
+export { RestError };
```

Comment:
Can we not combine this with the import statement? Say `export { RestError } from '@azure/ms-rest-js'`

Reply:
Fixed